### ISSUE TITLE
Segmentation fault on start

### ISSUE THREAD

**Issue by xRedFox** [TeaSpeak/TeaSpeak#208]
## System Information
***Operating System***: ArchLinux
***Operating System Type*** (x86|x64): x64
***TeaSpeak Version***: 1.1.37 beta 2

## Issue
***Reason for reporting this issue***:
Bug

***Detailed Description***:
Segmentation fuck every start without crash log.
[log_2018-06-29_15_05_32_general.zip](https://github.com/TeaSpeak/TeaSpeak/files/2149483/log_2018-06-29_15_05_32_general.zip)

**Comment by WolverinDEV** [comment]
Its seems for me that something is corrupt within your database.
Can you post a log here?

**Comment by M0V3** [comment]
![](https://i.gyazo.com/df1721503506f1a2058a3fc75b231509.png)

**Comment by WolverinDEV** [comment]
Fuck I thought Its a crash dump.
Well is there a crash dump?

And btw I have to rebuild the server again there's still a possible crash

**Comment by xRedFox** [comment]
I'm sorry but unfortunately there is no crash dump

**Comment by xRedFox** [comment]
@WolverinDEV I tried a few times to reproduce the crash locally and I came to the conclusion that if a user connects immediately after server loading (what actually happened to me when 30 people were reconnected after my maintenance) the server generates a segmentation fault.

![local](https://user-images.githubusercontent.com/19707386/42122560-e3a78720-7c43-11e8-99c6-190d3b2d9992.PNG)

It's not a problem of my database because the one used in the photo is the default one. I also confirm that the problem does not persist in the previous beta.
